package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build --configuration production=true",
+    "build": "ng build --configuration production",
```

Comment:
this value of "production" should match what's in angular.json.  not sure how this worked before this change with the latest version of angular.  if i'm somehow out of sync on angular versions or something, feel free to discard this change.